Formula/g2s.rb

```diff
@@ -0,0 +1,74 @@
+class G2s < Formula
+  desc "Toolbox for geostatistical simulations."
+  homepage "https://gaia-unil.github.io/G2S/"
+  url "https://github.com/GAIA-UNIL/g2s/archive/0acc033d06e7a3ac5ae4f13def02a43cec0adefe.tar.gz"
+  sha256 "7a95b04057bbdbbb11d37d4a6131d0b60981822c3e60e7986ea287b926b734ba"
+  license "GPL-3.0-only"
+  version "0.98.015"
```

Comment:
If you have control over the repo: https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff
> We frown on authors submitting their own work unless it is very popular.

Reply:
I read this section, and clearly miss this sentence. 

I have control on it, like the other member of the lab have control. In the same way popularity is very subjective. 
The idea is to go out of this niche situation, and provide an easy solution to install the package. Should I continue to invest time in correcting all the errors or it would be no go in all cases?

Reply:
Not for the moment, but I suppose we need to start somewhere 😄 
We generated .deb for the Debian manager, we will try to post them soon, but there we created in addition a x64 precompiled version with Intel compiler (that go up twice faster), and we are checking the licensing.

The project is running since 2017, and I was happy with the installation process. But over the last year, and the increase of the number of users, we get the number of requests of users to simplify the installation process.